node/forest_libp2p/src/blocksync/message.rs

```diff
@@ -22,11 +24,82 @@ pub struct BlockSyncRequest {
     pub options: u64,
 }
 
+impl BlockSyncRequest {
+    /// If a request expects blocks to be included in response.
+    pub fn include_blocks(&self) -> bool {
+        self.options == BLOCKS || self.options == BLOCKS_MESSAGES
+    }
+
+    /// If a request expects messages to be included in response.
+    pub fn include_messages(&self) -> bool {
+        self.options == MESSAGES || self.options == BLOCKS_MESSAGES
+    }
```

Comment:
These can be bit operations, `BLOCKS` is the least significant bit, and `MESSAGES` is the second least significant bit.

Can be something like:
```
    pub fn include_blocks(&self) -> bool {
        self.options & BLOCKS != 0
    }

    /// If a request expects messages to be included in response.
    pub fn include_messages(&self) -> bool {
        self.options & MESSAGES != 0
    }
```

Reply:
feels a little less readable imo :thinking: